src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -486,7 +489,11 @@ rct_window* window_scenery_open()
     gSceneryPlaceRotation = 0;
     gWindowSceneryPaintEnabled = 0; // repaint coloured scenery tool state
     gWindowSceneryEyedropperEnabled = false;
-    gWindowSceneryClusterEnabled = 0; // build cluster tool state
+    gWindowSceneryScatterEnabled = 0; // build cluster tool state
+    gWindowSceneryScatterWindow = nullptr;
+
+    gWindowSceneryScatterSize = 2;
+    gWindowSceneryScatterAmount = 10;
```

Comment:
Needs to be raised to 35 again, which is originally was.

Reply:
Why does it specifically need to be raised to 35 again?

Reply:
Because that was the old value. The scenery scatter window should default to the old parameters (number of trees and tool size).